### PR TITLE
Opt Out API status Endpoint Documentation

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@hodgestar Ready for review

File: docs/http_api.rst [hunk 1]
```diff
@@ -95,3 +95,26 @@ API Methods
 
    :statuscode 200: opt out deleted
    :statuscode 404: there's nothing to delete
+
+.. http:get:: /optouts/count
+
```
Comment (hodgestar) [inline docs/http_api.rst:100]: We should add a short description here of that this endpoint does.

File: docs/http_api.rst [hunk 2]
```diff
@@ -95,3 +95,26 @@ API Methods
...
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Vary: accept
+      Content-type: application/json
+
+      {
+        "opt_out_count": 3
+      }
+
```
Comment (hodgestar) [inline docs/http_api.rst:124]: We should add a `:statuscode 200:` section here.

**Comment by hodgestar** [comment]
I noticed that none of our endpoints document the `status` sections of the JSON return values. We should create a separate issue to add a general description of that near the top of the documentation and then update all the examples.

**Comment by hodgestar** [comment]
@TrevorOctober: Left a couple of comments.

**Comment by g1039** [comment]
@hodgestar  Ready for re-review

File: docs/http_api.rst [hunk 3]
```diff
@@ -95,3 +95,30 @@ API Methods
 
    :statuscode 200: opt out deleted
    :statuscode 404: there's nothing to delete
+
+.. http:get:: /optouts/count
+
+   Return the number of opt out
```
Comment (hodgestar) [inline docs/http_api.rst]: I think this should say `Return the number of opt outs.` -- note the fullstop and plural of opt outs.

**Comment by hodgestar** [comment]
One very minor comment, otherwise looks good.

**Comment by hodgestar** [comment]
@TrevorOctober Please wait for a :+1: before merging.

Retroactive :+1:. :)

**Pull request merged**